public/javascripts/running.js

```diff
@@ -168,39 +148,34 @@ function enableInteractive() {
 }
 
 function disableInteractive() {
-    window.hide("stop_button");
+    $("#stop_button").hide();
     sendCommand("disable_interactive_mode");
 }
 
 function sendCommand(command) {
-    var wid = window.testStatus.workerid;
+    var wid = testStatus.workerid;
```

Comment:
I guess that strictly speaking, ```window.testStatus``` has never been needed (```testStatus``` is enough), but most of the code use it that way because it looked to me in that time that it was the "clean" way of having global variables according to what coffeescript does (and since I have no idea of Javascript, I tend to trust experts).

Anyways, once again, it's inconsistent to "fix" (I'm actually completely unsure which option is best) only one.